eve/commands/cmd_weather.py

```diff
@@ -3,14 +3,8 @@
 import click
 
 from eve.service import svc_weather
-
-
-def convert_epoch_to(epoch, fmt):
-    return time.strftime(fmt, time.localtime(epoch))
-
-
-def is_around_midday(epoch):
-    return 11 <= int(convert_epoch_to(epoch, "%H")) <= 13
+from eve.utilities import utils as f
```

Comment:
I would also recommend just doing this as `import eve.utilities import utils` not need for the as